### PR TITLE
Issue #52 receiving error messages

### PR DESCRIPTION
**Pull request by frankyan1119**
**client.insert_rows_json(table, json_row)** keeps returning this error:

**"This json row did not load to BigQuery: [........] and threw this error: "**

### CODE REVIEW

**Comment by frankyan1119** [comment]
@googlebot I signed it!

File: 2-transform-data/bq_ingestion/extract_msgs_test.py [hunk 1]
```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
             got_json = em.convert_msg_to_json(test["msg"])
             self.assertEqual(want_json[key], got_json, "Convert message to json error")
 
+    
+    @mock.patch("google.cloud.storage.Client")
```
Comment (nyghtowl) [inline 2-transform-data/bq_ingestion/extract_msgs_test.py:730]: Change to google.cloud.bigquery.Client and mock.patch any other objects that are used in the function here like Table.

File: 2-transform-data/bq_ingestion/extract_msgs_test.py [hunk 2]
```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
...
+    def create_client_mock(self, Client):
```
Comment (nyghtowl) [inline 2-transform-data/bq_ingestion/extract_msgs_test.py:731]: pass in any variables you want to use in the mock before the objects you pass in. So if you want to simulate the table id you create, pass in the variable like `(self, table_id, Client)`

File: 2-transform-data/bq_ingestion/extract_msgs_test.py [hunk 3]
```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
...
+        client = Client()
+
```
Comment (nyghtowl) [inline 2-transform-data/bq_ingestion/extract_msgs_test.py:733]: Simulate any function calls and returns you need from the Client an other objects like `client.get_table(table_id)` would become `client.get_table.return_value = table_id`

File: 2-transform-data/bq_ingestion/extract_msgs_test.py [hunk 4]
```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
...
+        with open("table_schema.json") as f:
+            schema = json.load(f)
+        table_framework = em.bigquery.Table("some_project.some_dataset.123", schema=schema)
+        client.create_table(table_framework)
+
+        return client
+
+
     # TODO simulate load to BQ and test the components of this function esp errors
     def test_store_in_bigquery(self):
```
Comment (nyghtowl) [inline 2-transform-data/bq_ingestion/extract_msgs_test.py:743]: Thanks for keeping to the test structure.

**Review by nyghtowl — CHANGES_REQUESTED** [review]

**Comment by glasnt** [comment]
Closing, see #97

**Pull request closed without merging**